src/main/java/com/launchdarkly/client/LDUser.java

```diff
@@ -270,6 +270,17 @@ public Builder custom(String k, Number n) {
     }
 
     /**
+     * Add a {@link java.lang.Boolean}-valued custom attribute
+     * @param k the key for the custom attribute
+     * @param b the value for the custom attribute
+     * @return the builder
+     */
+    public Builder custom(String k, Boolean b) {
```

Comment:
This expands the API in a backward-compatible way, so the next version will be 0.9.0.

Also, we have a bunch of overloaded `custom` methods, to set values of different types, but you can only set a list of strings (because of type erasure, we can't have a method named `custom` that takes a list of numbers).

So, should we encode the type in the method names (eg, `customNumber(String, Number)` / `customNumbers(String, List<Number>)`, `customBool(String, Boolean)` / `customBools(String, List<Boolean>)`, `customString(String, String)` / `customStrings(String, List<String>)`)?  Or another idea?